### DISCUSSION TITLE
Should I remove scATAC-seq fragments when there are > 2

### DISCUSSION THREAD

**Discussion by Sas-cmd** [stuart-lab/signac#1094] [Q&A] [answered]
Hi all,

I was wondering what are your thoughts on removing duplicates that cellranger-arc might have missed? For example, looking at my  atac_fragments.tsv.gz file, I have fragments in certain locations for cells that are greater than 2 (up to 16) should I remove these fragments. 

If yes, how do I remove them? If no will the affect my downstream analysis, like DA?

Thank you!

**Comment by timoast** [comment, marked as answer]
If you're seeing regions with >2 overlapping fragments this could be caused by a couple of different things:
1. Cell doublets
2. Regions of the genome that are not properly represented in the genome assembly (mostly masked by the genomic blacklist sites)
3. Incorrect PCR duplicate collapsing
4. Differences in ploidy or copy number (endoreduplication, cell cycle,  structural variants)

There are different ways to deal with this depending on the underlying cause. [AMULET](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02469-x) is a method that identifies cell doublets using overlapping fragments. Genomic blacklist sites can be removed from downstream analysis. Incorrect PCR duplicate removal is more difficult to correct. Differences in ploidy or copy number are biological and I would not recommend doing anything to adjust for that.

**Comment by Sas-cmd** [comment]
Hi Tim,

Thank you for your response and recommendation for AMULET!

Can you also please recommend, methods for PCR duplicate removal?

Thank you!

**Reply by timoast** [reply]
PCR duplicate removal is already done to create the fragment file, you can see in the 5th column the PCR duplicate count for each fragment

**Reply by Sas-cmd** [reply]
Would it be correct then to remove all reads in the fragment file that has >1 read in the 5th column?

**Reply by timoast** [reply]
No definitely don’t do that

**Reply by Sas-cmd** [reply]
Can you please explain why that would not be the right thing to do? I am so terribly sorry for being such a pain but I am very confused as to why even after the 10x removal there is still such a high number of duplicates (higher number of fragments >2 counts). 

And how do I go about accounting for these duplicates?

**Reply by timoast** [reply]
The fragment file is already de-duplicated, the PCR duplicate count number in the 5th column is not used in the analysis, it just tells you the sequencing saturation. There's no reason to discard fragments based on that.
